siliconcompiler/report/streamlit_viewer.py

```diff
@@ -335,7 +326,7 @@ def manifest_module(chip, manifest, ui_width, max_num_of_keys_to_show=20,
                               help='Click here to see the JSON before it was made more readable'):
             manifest_to_show = chip.schema.cfg
         else:
-            manifest_to_show = manifest
+            manifest_to_show = MANIFEST
```

Comment:
where does this come from?

Reply:
It is set in the beginning and remains unchanged, so I made it a constant that is set at the beginning of running the dashboard.

Reply:
lower case